src/SprykerEco/Yves/Payone/Dependency/Client/PayoneToStoreClientInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Yves\Payone\Dependency\Client;
+
+interface PayoneToStoreClientInterface
+{
+    /**
```

Comment:
* In the bridge methods the specification and `@api` is not needed.

Reply:
++